src/theming/sap_fiori_3.scss

```diff
@@ -1,3 +1,19 @@
 :root {
   --sapInverted_Object_Border_Width: 0;
+
+  /* Rating Indicator */
+  // default mode
+  --sapRating_Indicator_Selected_Icon_Color: #d08014;
+  --sapRating_Indicator_Selected_Hover_Opacity: 0.9;
+  --sapRating_Indicator_Unselected_Icon_Color: #89919a;
+  // Readonly mode
+  --sapRating_Indicator_Readonly_Selected_Color: #d08014;
```

Comment:
Can you use the CSS variables instead of hardcoding the values?
for the colors bellow

Reply:
fixed